src/www/system_general.php

```diff
@@ -46,6 +46,7 @@
     $pconfig['hostname'] = $config['system']['hostname'];
     $pconfig['language'] = $config['system']['language'];
     $pconfig['prefer_ipv4'] = isset($config['system']['prefer_ipv4']);
+    $pconfig['prefer_dpinger'] = isset($config['system']['prefer_dpinger']);
```

Comment:
note to self: should be in misc, along with the default gateway switching option which is currently under firewall

(it's not a basic setting)